auth-web/src/components/auth/staff/continuation-in/StaffContinuationInTable.vue

```diff
@@ -0,0 +1,310 @@
+                      :key="i"
+                      class="font-weight-bold"
+                    >
+                      {{ header.text }}
+                    </th>
+                  </tr>
+
+                  <!-- Second row has search boxes. Search and filter to be implemented -->
+                  <tr class="header-row-2 mt-2 px-2">
+                    <th
+                      v-for="(header, i) in headerAccounts"
+                      :key="getIndexedTag('find-header-row2', i)"
+                      :scope="getIndexedTag('find-header-col2', i)"
+                    >
+                      <v-text-field
+                        v-if="!['action'].includes(header.value)"
+                        :id="header.value"
+                        input
+                        type="search"
+                        autocomplete="off"
+                        class="text-input-style"
+                        filled
+                        :placeholder="header.text"
+                        dense
+                        hide-details="auto"
+                      />
+                    </th>
+                  </tr>
+                </thead>
+              </template>
+
+              <!-- Item Actions -->
+              <template #[`item.action`]="{ item }">
+                <div class="actions text-right">
+                  <span class="open-action">
+                    <v-btn
+                      color="primary"
+                      :class="['open-action-btn', { active: isActiveAccounts }]"
+                      :data-test="getIndexedTag('view-account-button', item.id)"
+                      @click="view()"
+                    >
+                      {{ getButtonLabel(item.status) }}
+                    </v-btn>
+                  </span>
+                </div>
+              </template>
+            </v-data-table>
+          </transition>
+        </v-col>
+      </v-row>
+    </v-form>
+  </div>
+</template>
+
+<script lang="ts">
+import { defineComponent, reactive } from '@vue/composition-api'
+import BusinessServices from '@/services/business.services'
+
+export default defineComponent({
+  name: 'StaffAccountsTable',
+  setup () {
+    const state = reactive({
+      businesses: [],
+      headerAccounts: [
+        { text: 'Date Submitted', value: 'date' },
+        { text: 'NR Number', value: 'nrNumber' },
+        { text: 'Identifying Number', value: 'businessIdentifier' },
+        { text: 'Completing Party', value: 'completingParty' },
+        { text: 'Status', value: 'status' },
+        { text: 'Actions', value: 'action' }
+      ],
+      isTableLoading: false,
+      paginationOptions: [5, 10, 15, 20]
+    })
+
+    function getButtonLabel (status) {
+      const reviewStates = ['Awaiting Review', 'Resubmitted']
+      return reviewStates.includes(status) ? 'Review' : 'View'
+    }
+
+    function view () {
+      // To be updated: route to Continuation Authorization Review page
+      // root.$router.push()
```

Comment:
Here, route to `/staff/continuation-review/<review-id>`. It will show "page not found" until my code is in place and that's OK.